packages/client/src/components/UserProfileButtonBlock/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { UserFormLinkButton } from '../UserFormLinkButton'
+import style from './index.module.scss'
+
+type TUserProfileButtonBlock = {
+  logOutCallback: () => void
+  userDataChangeCallback: () => void
+  passwordChangeCallback: () => void
```

Comment:
И также нет необходимости называть его callback, это и так ясно. Лучше просто назови как есть (это распространённая практика)
```suggestion
  logOutClick: () => void
  userDataChangeClick: () => void
  passwordChangeClick: () => void // или просто   passwordChange
```